facebook/src/main/java/com/facebook/GraphRequest.java

```diff
@@ -1706,7 +1706,8 @@ final static void serializeToUrlConnection(
 
         OutputStream outputStream = null;
         try {
-            outputStream = new BufferedOutputStream(connection.getOutputStream());
+            outputStream = connection.getOutputStream();
```

Comment:
Agree, as far as BufferedOutputStream's constructor doesn't throw exception, it's more precaution, rather than actually required.